Library/Formula/subversion.rb

```diff
@@ -30,11 +24,12 @@ class Subversion < Formula
 
   resource "serf" do
     url "https://serf.googlecode.com/svn/src_releases/serf-1.3.8.tar.bz2", :using => :curl
+    mirror "https://fossies.org/linux/www/serf-1.3.8.tar.bz2"
     sha256 "e0500be065dbbce490449837bb2ab624e46d64fc0b090474d9acaa87c82b2590"
   end
 
   depends_on "pkg-config" => :build
-  depends_on :apr => :build
+  depends_on "apr" => :build
```

Comment:
This will do the opposite and *always* enforce Homebrew's `apr` :wink:. I don't think that's the worst thing in the world personally, but we try hard to avoid unnecessary system dupes and this would be one.